packages/polyfill/index.ts

```diff
@@ -21,11 +21,16 @@ import 'whatwg-fetch';
 // Promise
 import 'promise-polyfill/lib/polyfill';
 
-// ES6
+// ES Stable
 import 'core-js/features/array/find';
 import 'core-js/features/array/find-index';
+import 'core-js/features/array/from';
 import 'core-js/features/object/assign';
+import 'core-js/features/object/entries';
+import 'core-js/features/object/values';
 import 'core-js/features/string/starts-with';
 import 'core-js/features/string/repeat';
 import 'core-js/features/symbol';
 import 'core-js/features/symbol/iterator';
+import 'core-js/features/map';
+import 'core-js/features/set';
```

Comment:
@Feiyang1 Since we no longer ship these Polyfills by default, are changes like this considered breaking changes?

@mmermerkaya What is the size impact of these Polyfills?

Reply:
They shouldn't be considered as breaking changes.